src/sources.js

```diff
@@ -35,7 +35,7 @@ export default tracker => ({ dispatch }) => next => action => {
       });
     };
 
-    runContextual(run);
+    run(register);
```

Comment:
See the comment in [`run.js`](https://github.com/samybob1/meteor-redux-middlewares/pull/15/files#r94862359),  this could just be `setTimeout`